Jenkinsfile

```diff
@@ -96,7 +96,7 @@ pipeline {
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -pl $ITESTS,!$ITCORE -amd -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
```

Comment:
✏️ Why is `-amd` needed here?

Reply:
Here's the description of the `-amd` flag from the Maven docs:
```
-amd, --also-make-dependents
        If project list is specified, also build projects that depend on projects on the list
```
If you just run `mvn install -pl distribution/test/itests`, it will only build the itests project, not the children. The children are dependent projects, so adding `-amd` allows us to build the whole project tree under the itests module without specifying each module individually